tensorflow/core/kernels/data/captured_function.cc

```diff
@@ -813,9 +813,9 @@ InstantiatedCapturedFunction::InstantiatedCapturedFunction(
       captured_func_(captured_func),
       is_multi_device_(is_multi_device) {}
 
-Status InstantiatedCapturedFunction::Run(IteratorContext* ctx,
-                                         std::vector<Tensor>&& args,
-                                         std::vector<Tensor>* rets) const {
+Status InstantiatedCapturedFunction::Run(
```

Comment:
Could you apply a similar change to `RunWithBorrowedArgs`? Don't worry about `RunInstantiated` (as that one is only invoked in one place during iterator destruction). Thanks.

Reply:
Done